src/figuresClasses.ts

```diff
@@ -1,11 +1,103 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'rectangle' | 'circle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  constructor(
+    a: number,
+    b: number,
+    c: number,
+    public shape: 'triangle',
+    public color: 'red' | 'green' | 'blue',
+  ) {
+    this.a = a;
+    this.b = b;
+    this.c = c;
```

Comment:
[CODE KNOWLEDGE] - You do not need to manually assign constructor parameters to class properties if you use the TypeScript access modifier syntax. Since `shape` and `color` are already using this syntax, you can do the same for `a`, `b`, and `c` to simplify your code.